app/src/main/java/fr/lille1/univ/android/architecture/btrshop/detailsproduct/domain/model/Offer.java

```diff
@@ -0,0 +1,103 @@
+package fr.lille1.univ.android.architecture.btrshop.detailsproduct.domain.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+import java.util.Date;
+
+/**
+ * Created by denis on 03/11/2016.
+ */
+
+public class Offer implements Serializable {
```

Comment:
This class has been created to match with the API and the schema "offer" that appears in the schema "product" https://schema.org/Offer https://schema.org/Product